sql/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -214,6 +214,18 @@ object Connect {
       .toSequence
       .createWithDefault(Nil)
 
+  val CONNECT_ML_BACKEND_CLASSES =
+    buildConf("spark.connect.ml.backend.classes")
```

Comment:
Hi @grundprinzip, Do you think  the conf name "spark.connect.ml.backend.classes" is ok? I'm free to change it according to your suggestions.

Reply:
I think backend classes is fine because it's not really a plugin.